src/ixdat/techniques/ms.py

```diff
@@ -424,18 +462,8 @@ def gas_flux_calibration_curve(
             periods.
         TODO: automatically recognize the pressure from measurement (if available)
         """
-        if not plugins.use_si_quant:
-            raise QuantificationError(
-                "`MSMeasurement.gas_flux_calibration` only works when using an "
-                "external MS quantification package "
-                "(`ixdat.options.use_si_quant = True`). "
-                "For native ixdat MS quantification, `gas_flux_calibration` has to be"
-                "called from an instance of `MSInlet`."
-            )
-        Chip = plugins.si_quant.Chip
-        CalPoint = plugins.si_quant.CalPoint
-
-        chip = chip or Chip()
+        # prepare three lists to loop over to determine molecule flux in the
```

Comment:
Line 465 and 466 looks identical to 468 and 469

Reply:
Yes. fixed.